dist-workspace.toml

```diff
@@ -13,3 +13,8 @@ installers = []
 targets = ["aarch64-apple-darwin", "x86_64-apple-darwin", "x86_64-unknown-linux-gnu", "x86_64-unknown-linux-musl", "x86_64-pc-windows-msvc"]
 # Which actions to run on pull requests
 pr-run-mode = "plan"
```

Comment:
I believe this is the default; do we need this in here? Or it useful when dorking around with this config?

Reply:
Right, I had changed it to `update` to confirm that builds worked, then reverted it back to the original value in 6f5a1ba389f3bbedaac43526e8a3a2186ce5746a